code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -183,96 +183,96 @@
 	allowed_roles = list("Research Director")
 */
 /datum/gear/uniform/dept/undercoat/qm
-	display_name = "Teshari Quarter Master Undercoat"
+	display_name = "quartermaster undercoat (Teshari)"
 	path = /obj/item/clothing/under/seromi/undercoat/jobs/qm
 	allowed_roles = list("Quartermaster")
 
 /datum/gear/uniform/dept/undercoat/command
-	display_name = "Teshari Command Undercoat"
+	display_name = "command undercoat (Teshari)"
 	path = /obj/item/clothing/under/seromi/undercoat/jobs/command
 	allowed_roles = list("Colony Director","Head of Personnel","Head of Security","Chief Engineer","Chief Medical Officer")
```

Comment:
There's not anything for Research Director?

Reply:
I don't quite remember who sprited all of these, but there are several jobs that did not get a unique coat or cape. I am not a spriter, this PR just fixes ones that do exist and had the icons broken. Although it is weird that RD doesn't get command coat, gonna have to fix that.

Reply:
👍